questions.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+var questions = [],
+    fs = require('fs'),
+	JSON5 = require('json5'),
+    correctNameEnding,
+    jsonData;
+
+var originCountries = ['Україна', 'Англія', 'Франція', 'Іспанія', 'Німеччина', 'Чехія', 'Молдова', 'Уельс', 'Китай', 'Азербайджан', 'Чилі'];
+
+//fs.readFile('./public/freebase_countries.json', 'utf8', function(err, data){
+//    jsonData = JSON5.parse(data);
+//    getDataFromJson(jsonData, questions);
+//    console.log(questionsJS.questions);
+//});
+
+for(var i in originCountries) {
+    correctNameEnding = changeCountryNameEnding(originCountries[i]);
+    questions.push(correctNameEnding);
+}
```

Comment:
Can you read about `map` function and see if you can use it?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map